.github/workflows/tests.yml

```diff
@@ -18,7 +18,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        rails-version: [5.2, 6.1]
+        ruby-version: ["2.7", "3.0", "3.1"]
```

Comment:
Hello @excid3!

Thank you a lot for this contribution, this will help us to grow with even more confidence!

I'm afraid about raising the compatibility versions here without starting from the meilisearch-ruby first, so I will suggest you add the `2.6` as well since the `2.6` was the default one before your implementation 😃

Probably after the `2.6` EOL date, we will wait some time to drop the support from ruby 2.6 to let the community upgrade their apps.

Reply:
Yeah! Of course. Just added. 👍